docs/modules/ROOT/pages/reference/commands.adoc

```diff
@@ -41,6 +41,7 @@ check with the user whether they really want to remove the items listed above.
 
 This command creates a new component repository under `dependencies/` in Commodore's working directory.
 The component repository is created using a Cookiecutter template which provides a skeleton for writing a new component.
+The command requires the argument `COMPONENT_NAME` to match the regular expression `^[a-z][a-z0-9-_]+[a-z0-9]$`.
```

Comment:
Can we bring the help output and the documentation in sync? The CLI help output uses the term `SLUG`. I assume this was missed the last time we touched this.